kubetest/e2e.go

```diff
@@ -37,6 +37,9 @@ func run(deploy deployer, o options) error {
 	os.Setenv("KUBE_CONFIG_FILE", "config-test.sh")
 	// force having batch/v2alpha1 always on for e2e tests
 	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
+	// force having apps/v1beta2 always on for e2e tests
+	// TODO: remove it once we enable apps/v1beta2 by default
+	os.Setenv("KUBE_RUNTIME_CONFIG", "apps/v1beta2=true")
```

Comment:
I cannot find `test_context.go` in this repo. Where's that file and what flag be set?

Is it okay if I instead change L39 into:

```go 
os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true,apps/v1beta2=true")
```
and then revert this change once apps/v1beta2 is enabled by default? We only need this temporarily (during 1.8 implementation stage) so that we can use this new version in e2e.